flocker/control/_model.py

```diff
@@ -552,7 +553,8 @@ class NodeState(PRecord):
     """
     The current state of a node.
 
-    :ivar unicode hostname: The hostname of the node.
+    :ivar UUID uuid: The node's UUID.
+    :ivar unicode hostname: The IP of the node.
```

Comment:
Notice that all I'm doing is changing the docstring to be more accurate, no actual code changes will result.